src/Stan.hs

```diff
@@ -35,7 +37,15 @@ runStan StanArgs{..} = do
     -- create Map from filepathes to sources.
     let sourcesMap = fromList $ map (\HieFile{..} -> (hie_hs_file, hie_hs_src)) hieFiles
     -- create extensions map
-    extensionsMap <- getPackageExtentionsBySources "stan.cabal" sourcesMap
+    extensionsMap <- findCabalFile >>= \case
+        Just cabal -> do
+            infoMessage $ "Using the following .cabal file: " <> toText cabal <> "\n"
+            getPackageExtentionsBySources cabal sourcesMap
+        -- if cabal file is not found, pass the empty map instead
```

Comment:
Good solution for now 👍  I think in the future we can support work even without the `.cabal` file by collecting extensions only in modules. So we somehow need to parse each module separately, if the `.cabal` file not found. I don't see a straight solution at the moment, but should be possible.  Like, with printing the warning that cabal file not found and that the list of extensions might not be complete.

Reply:
Excellent point! I've created the issue for that: https://github.com/kowainik/stan/issues/82